core/domain/classifier_services.py

```diff
@@ -55,7 +56,9 @@ def classify(state, answer):
     response = None
 
     if interaction_instance.is_string_classifier_trainable:
-        response = classify_string_classifier_rule(state, normalized_answer)
+        response = classify_string_classifier_rule(state, normalized_answer,
+                                                   exp_id, exp_version,
```

Comment:
When indenting from an open parenthesis ('('), prefer indenting by 4 rather than indenting from the position of the parenthesis.